### ISSUE TITLE
Process.Start Cannot find file when UseShellExecute is set to false

### ISSUE THREAD

**Issue by abjrcode** [dotnet/corefx#34984]
I am using `Process.Start()` to start two different processes. In my case the two processes are `docker` and `aws`, the latter which is the AWS command line utilities.  
`Process.Start` is able to run the `docker` process successfully regardless of the `UseShellExecute` value
It is not able to run `aws` command though unless I specify `UseShellExecute = true`

I have noticed this behavior on both the latest version of .NET Core and latest version of the .NET framework

Both process paths are listed in the `PATH` environment variable.
I am running Windows 10 version 1808 (build 17763.194)

DotNetCore 2.2 and .NET Framework 4.7.2 both have this behavior
Only difference I noticed between the two executables is that `docker` is an `exe` application while `aws` runs a `cmd` script, should that matter?

**Comment by stephentoub** [comment]
> while aws runs a cmd script, should that matter?

Yes, it matters, that's what UseShellExecute=true is there to support. It's the shell that understands how to run scripts.

**Comment by abjrcode** [comment]
That actually used to work somehow before, I don't know what broke it though
What you are saying makes sense but I can't explain how it used to work before

**Comment by stephentoub** [comment]
> That actually used to work somehow before

Not sure how from the .NET side of things.  Maybe `aws` used to be an executable and changed in an SDK update or something to be a script?

Regardless, it sounds like this are behaving the way they're supposed to behave in .NET, for better or worse.

> but how can you read the standard output and error streams of aws if you need a shell instance to run them, as far as I know from docs that is not possible correct?

You can't redirect stdin/out/err when UseShellExecute is true.

**Comment by abjrcode** [comment]
> Not sure how from the .NET side of things. Maybe aws used to be an executable and changed in an SDK update or something to be a script?

I am thinking the same

Thanks a lot for your help
